src/main/java/duke/parser/Parser.java

```diff
@@ -1,3 +1,18 @@
+package duke.parser;
+
+import duke.command.AddCommand;
```

Comment:
Similar to previous comments, maybe the various command types can be separate methods within the Command Class, which reduces the need for having to repeat the isExit() method?

Reply:
Thanks for the suggestion! Agree that having to repeat the isExit() method is redundant and that the various commands can be rewritten in a more succinct manner.